src/node_report.cc

```diff
@@ -782,6 +790,17 @@ static void PrintRelease(JSONWriter* writer) {
   writer->json_objectend();
 }
 
+static void PrintRequestInfo(JSONWriter* writer, Environment* env) {
+  for (ReqWrapBase* req_wrap : *env->req_wrap_queue()) {
+    AsyncWrap* w = req_wrap->GetAsyncWrap();
+    if (w->persistent().IsEmpty()) continue;
+    writer->json_start();
+    writer->json_keyvalue("type", w->MemoryInfoName().c_str());
+    // We can add more info here
```

Comment:
What additional information can we add here? The request type alone is not useless, but also not terribly helpful.

Reply:
No additional information is available currently, new fields can be added to the object in the future.